### PR TITLE
types.go is no longer a thing

### PR DESCRIPTION
**Pull request by Mingling94**
So I reflected that in the documentation as it threw me for a
small loop when I looked for it.

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -569,7 +569,7 @@ struct {
 	Transactions []consensus.Transaction
 }
 ```
-Please see consensus/types.go for a more detailed explanation on what a
+Please see consensus/types/transactions.go for a more detailed explanation on what a
```
Comment (lukechampine) [inline doc/API.md]: wrap this to 80 chars

File: doc/Consensus.md [hunk 2]
```diff
@@ -13,7 +13,8 @@ If you wish to know more about the protocol and the consensus rules, you should
 read the documentation in the following order:
 
 - Consensus.md
-- consensus/types.go
+- consensus/types
+    - Here's the [auto-generated go-documentation](https://godoc.org/github.com/NebulousLabs/Sia/types)
```
Comment (lukechampine) [inline doc/Consensus.md]: change this to:
```markdown
- consensus/types -- refer to [godoc](https://godoc.org/github.com/NebulousLabs/Sia/types)
```

**Comment by DavidVorick** [comment]
for the record, you don't need to open a new pull request when you update your branch. The changes you make to the branch will automatically go to the pull request.

I'm not sure why you closed the other one.

**Comment by Mingling94** [comment]
Hah, I failed to use git properly with the other one and included a few non-purely-documentation commits.

**Pull request merged**